lua/apisix/plugins/tcp-logger.lua

```diff
@@ -59,21 +59,21 @@ local function send_tcp_data(conf, log_message)
     local sock, soc_err = tcp()
 
     if not sock then
-        return nil, "failed to init the socket" .. soc_err
+        return false, "failed to init the socket" .. soc_err
     end
 
-    sock:settimeout(conf.timeout * 1000)
+    sock:settimeout(conf.timeout)
 
     local ok, err = sock:connect(conf.host, conf.port)
     if not ok then
-        return nil, "failed to connect to TCP server: host[" ..
+        return false, "failed to connect to TCP server: host[" ..
             conf.host .. "] port[" .. tostring(conf.port) .. "] err: " .. err
```

Comment:
Bad indentation, here is good style:

```lua
        return false, "failed to connect to TCP server: host[" ..
                       conf.host .. "] port[" .. tostring(conf.port) .. "] err: " .. err
```

Reply:
please fix other similar points

Reply:
Hi, @membphis  does the following like should be also intended like this?
before:
```lua
err_msg = "failed to send data to TCP server: host[" ..
            conf.host .. "] port[" .. tostring(conf.port) .. "] err: " .. err
```
after:
```lua
err_msg = "failed to send data to TCP server: host[" ..
                  conf.host .. "] port[" .. tostring(conf.port) .. "] err: " .. err
```

Reply:
you are right. the new style is clearer.